$MirOS: ports/games/bsdgames/patches/patch-fortune_datfiles_fortunes,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- fortune/datfiles/fortunes.orig	Thu Jan  1 00:00:00 1970
+++ fortune/datfiles/fortunes	Thu Feb 22 21:51:06 2007
@@ -1783,7 +1783,7 @@ Ask your boss to reconsider -- it's so d
 for an answer.
 %
 "Asked by reporters about his upcoming marriage to a forty-two-year-old
-woman, director Roman Polanski told reporters, `The way I look at it,
+woman, director Roman Polanski told reporters, 'The way I look at it,
 she's the equivalent of three fourteen-year-olds.'"
 		-- David Letterman
 %
@@ -1900,7 +1900,7 @@ taxes.
 Basic is a high level languish.
 APL is a high level anguish.
 %
-"BASIC is the Computer Science equivalent of `Scientific Creationism'."
+"BASIC is the Computer Science equivalent of 'Scientific Creationism'."
 %
 BASIC, n.:
 	A programming language.  Related to certain social diseases in
@@ -2169,7 +2169,7 @@ Boys will be boys, and so will a lot of 
 Brace yourselves.  We're about to try something that borders on the
 unique: an actually rather serious technical book which is not only
 (gasp) vehemently anti-Solemn, but also (shudder) takes sides.  I tend
-to think of it as `Constructive Snottiness.'
+to think of it as 'Constructive Snottiness.'
 		-- Mike Padlipsky, Foreword to "Elements of Networking Style"
 %
 Bradley's Bromide:
@@ -2479,7 +2479,7 @@ eventually it began to sink in.  Finally
 the word "canonical" in jargon-like fashion without thinking.
 	Steele: "Aha!  We've finally got you talking jargon too!"
 	Stallman: "What did he say?"
-	Steele: "He just used `canonical' in the canonical way."
+	Steele: "He just used 'canonical' in the canonical way."
 %
 CAPRICORN (Dec 23 - Jan 19)
 	You are conservative and afraid of taking risks.  You don't do much
@@ -3014,7 +3014,7 @@ Death is life's way of telling you you'v
 %
 Death is Nature's way of recycling human beings.
 %
-Death is nature's way of saying `Howdy'.
+Death is nature's way of saying 'Howdy'.
 %
 Death is nature's way of telling you to slow down.
 %
@@ -4473,8 +4473,8 @@ Future looks spotty.  You will spill sou
 %
 G. B. Shaw to William Douglas Home: "Go on writing plays, my boy.  One
 of these days a London producer will go into his office and say to his
-secretary, `Is there a play from Shaw this morning?' and when she says
-`No,' he will say, `Well, then we'll have to start on the rubbish.' And
+secretary, 'Is there a play from Shaw this morning?' and when she says
+'No,' he will say, 'Well, then we'll have to start on the rubbish.' And
 that's your chance, my boy."
 %
 Garbage In -- Gospel Out.
@@ -4858,11 +4858,11 @@ Grandpa Charnock's Law:
 Gravity is a myth: the Earth sucks.
 %
 Gray's Law of Programming:
-	`_n+1' trivial tasks are expected to be accomplished in the same
-	time as `_n' tasks.
+	'_n+1' trivial tasks are expected to be accomplished in the same
+	time as '_n' tasks.
 
 Logg's Rebuttal to Gray's Law:
-	`_n+1' trivial tasks take twice as long as `_n' trivial tasks.
+	'_n+1' trivial tasks take twice as long as '_n' trivial tasks.
 %
 Great minds run in great circles.
 %
@@ -5292,7 +5292,7 @@ but broad satiric vistas do not open up.
 		-- John Carrol, San Francisco Chronicle
 %
 "Here's something to think about:  How come you never see a headline like
-`Psychic Wins Lottery'?"
+'Psychic Wins Lottery'?"
 		-- Jay Leno
 %
 Heuristics are bug ridden by definition.  If they didn't have bugs,
@@ -5315,7 +5315,7 @@ experience and training, there's no reas
 of this thing with at least a cabin cruiser.
 
 "Remember, at the Preston A. Mantis Consumers Retail Law Outlet, our
-motto is:  `It is very difficult to disprove certain kinds of pain.'"
+motto is:  'It is very difficult to disprove certain kinds of pain.'"
 		-- Dave Barry, "Pain and Suffering"
 %
 Hier liegt ein Mann ganz ohnegleich;
@@ -5595,7 +5595,7 @@ I am not now, and never have been, a gir
 I am not now, nor have I ever been, a member of the demigodic party.
 		-- Dennis Ritchie
 %
-I am not sure what this is, but an `F' would only dignify it.
+I am not sure what this is, but an 'F' would only dignify it.
 		-- English Professor
 %
 I am ready to meet my Maker.  Whether my Maker is prepared for the
@@ -5788,11 +5788,11 @@ people waiting to abuse me.
 I don't know anything about music.  In my line you don't have to.
 		-- Elvis Presley
 %
-	"I don't know what you mean by `glory,'" Alice said
+	"I don't know what you mean by 'glory,'" Alice said
 	Humpty Dumpty smiled contemptuously.  "Of course you don't --
-till I tell you.  I meant `there's a nice knock-down argument for
+till I tell you.  I meant 'there's a nice knock-down argument for
 you!'"
-	"But glory doesn't mean `a nice knock-down argument,'" Alice
+	"But glory doesn't mean 'a nice knock-down argument,'" Alice
 objected.
 	"When I use a word," Humpty Dumpty said, in a rather scornful
 tone, "it means just what I choose it to mean -- neither more nor
@@ -5907,7 +5907,7 @@ I have a very firm grasp on reality!  I 
 any time!
 %
 I have come up with a sure-fire concept for a hit television show,
-which would be called `A Live Celebrity Gets Eaten by a Shark'.
+which would be called 'A Live Celebrity Gets Eaten by a Shark'.
 		-- Dave Barry, "The Wonders of Sharks on TV"
 %
 I have discovered the art of deceiving diplomats.
@@ -6047,8 +6047,8 @@ I profoundly believe it takes a lot of p
 		-- William F. Buckley
 %
 	"I quite agree with you," said the Duchess; "and the moral of
-that is -- `Be what you would seem to be' -- or, if you'd like it put
-more simply -- `Never imagine yourself not to be otherwise than what it
+that is -- 'Be what you would seem to be' -- or, if you'd like it put
+more simply -- 'Never imagine yourself not to be otherwise than what it
 might appear to others that what you were or might have been was not
 otherwise than what you had been would have appeared to them to be
 otherwise.'"
@@ -6220,8 +6220,8 @@ safer, the beer stronger, the food cheap
 warmer in the winter, and happier in the summer.
 		-- Brendan Behan
 %
-"I want to buy a husband who, every week when I sit down to watch `St.
-Elsewhere', won't scream, `FORGET IT, BLANCHE ... IT'S TIME FOR "HEE
+"I want to buy a husband who, every week when I sit down to watch 'St.
+Elsewhere', won't scream, 'FORGET IT, BLANCHE ... IT'S TIME FOR "HEE
 HAW"!!'"
 		-- Berke Breathed, "Bloom County"
 %
@@ -6289,7 +6289,7 @@ I went to the race track once and bet on
 it took seven others to beat him!
 %
 I wish there was a knob on the TV to turn up the intelligence.
-There's a knob called `brightness', but it doesn't seem to work.
+There's a knob called 'brightness', but it doesn't seem to work.
 		-- Gallagher
 %
 I wouldn't recommend sex, drugs or insanity for everyone, but they've
@@ -6307,7 +6307,7 @@ to undo it."
 snore."
 %
 "I'd love to go out with you, but I never go out on days that end in
-`Y.'"
+'Y.'"
 %
 "I'd love to go out with you, but I want to spend more time with my
 blender."
@@ -7273,11 +7273,11 @@ moving automobile.
 %
 [In the 60's] there was madness in any direction, at any hour ...  You
 could strike sparks anywhere.  There was a fantastic universal sense
-that whatever we were doing was `right', that we were winning ...
+that whatever we were doing was 'right', that we were winning ...
 
 And that, I think, was the handle -- the sense of inevitable victory
 over the forces of Old and Evil.  Not in any mean or military sense; we
-didn't need that.  Our energy would simply `prevail'.  There was no
+didn't need that.  Our energy would simply 'prevail'.  There was no
 point in fighting -- on our side or theirs.  We had all the momentum;
 we were riding the crest of a high and beautiful wave ....
 
@@ -7735,7 +7735,7 @@ It's better to be wanted for murder than
 "It's easier said than done."
 
 ... and if you don't believe it, try proving that it's easier done than
-said, and you'll see that "it's easier said that `it's easier done than
+said, and you'll see that "it's easier said that 'it's easier done than
 said' than it is done", which really proves that "it's easier said than
 done".
 %
@@ -7936,7 +7936,7 @@ of the few remaining bits of your brain 
 Just remember: when you go to court, you are trusting your fate to
 twelve people that weren't smart enough to get out of jury duty!
 %
-`Just the place for a Snark!' the Bellman cried,
+'Just the place for a Snark!' the Bellman cried,
 	As he landed his crew with care;
 Supporting each man on the top of the tide
 	By a finger entwined in his hair.
@@ -8097,7 +8097,7 @@ Lassie filed the applications for.
 %
 Last night, I came home and realized that everything in my apartment
 had been stolen and replaced with an exact duplicate.  I told this to
-my friend -- he said, `Do I know you?'
+my friend -- he said, 'Do I know you?'
 		-- Steven Wright
 %
 Last week a cop stopped me in my car.  He asked me if I had a police
@@ -9118,7 +9118,7 @@ villagers gathered around to hear what h
 said Nasrudin, "I only want to say that the King spoke to me."  All the
 villagers but the stupidest ran off to spread the wonderful news.  The
 remaining villager asked, "What did the King say to you?"  "What he
-said -- and quite distinctly, for everyone to hear -- was `Get out of
+said -- and quite distinctly, for everyone to hear -- was 'Get out of
 my way!'" The simpleton was overjoyed; he had heard words actually
 spoken by the King, and seen the very man they were spoken to.
 %
@@ -9381,7 +9381,7 @@ No violence, gentlemen -- no violence, I
 the furniture!
 		-- Sherlock Holmes
 %
-"No, `Eureka' is Greek for `This bath is too hot.'"
+"No, 'Eureka' is Greek for 'This bath is too hot.'"
 		-- Dr. Who
 %
 Nobody can be exactly like me.  Sometimes even I have trouble doing it.
@@ -9715,7 +9715,7 @@ On the subject of C program indentation:
 	indented six feet downward and covered with dirt."
 		-- Blair P. Houghton
 %
-On two occasions I have been asked [by members of Parliament!], `Pray,
+On two occasions I have been asked [by members of Parliament!], 'Pray,
 Mr.  Babbage, if you put into the machine wrong figures, will the right
 answers come out?'  I am not able rightly to apprehend the kind of
 confusion of ideas that could provoke such a question.
@@ -10791,7 +10791,7 @@ Real computer scientists don't program i
 in anything less portable than a number two pencil.
 %
 Real computer scientists don't write code.  They occasionally tinker
-with `programming systems', but those are so high level that they
+with 'programming systems', but those are so high level that they
 hardly count (and rarely count accurately; precision is for
 applications.)
 %
@@ -11314,7 +11314,7 @@ Serving coffee on aircraft causes turbul
 %
 	"Seven years and six months!"  Humpty Dumpty repeated
 thoughtfully.  "An uncomfortable sort of age.  Now if you'd asked MY
-advice, I'd have said `Leave off at seven' -- but it's too late now."
+advice, I'd have said 'Leave off at seven' -- but it's too late now."
 	"I never ask advice about growing,"  Alice said indignantly.
 	"Too proud?" the other enquired.
 	Alice felt even more indignant at this suggestion.  "I mean,"
@@ -11376,7 +11376,7 @@ were bad.
 She missed an invaluable opportunity to give him a look that you could
 have poured on a waffle.
 %
-She said, `I know you ... you cannot sing'.  I said, `That's nothing,
+She said, 'I know you ... you cannot sing'.  I said, 'That's nothing,
 you should hear me play piano.'
 		-- Morrisey
 %
@@ -12324,7 +12324,7 @@ The economy depends about as much on eco
 weather forecasters.
 		-- Jean-Paul Kauffmann
 %
-"The eleventh commandment was `Thou Shalt Compute' or `Thou Shalt Not
+"The eleventh commandment was 'Thou Shalt Compute' or 'Thou Shalt Not
 Compute' -- I forget which."
 		-- Epigrams in Programming, ACM SIGPLAN Sept. 1982
 %
@@ -12861,16 +12861,16 @@ The moving cursor writes, and having wri
 %
 ... the MYSTERIANS are in here with my CORDUROY SOAP DISH!!
 %
-	"... The name of the song is called `Haddocks' Eyes'!"
+	"... The name of the song is called 'Haddocks' Eyes'!"
 	"Oh, that's the name of the song, is it?" Alice said, trying to
 feel interested.
 	"No, you don't understand," the Knight said, looking a little
-vexed.  "That's what the name is called.  The name really is, `The Aged
+vexed.  "That's what the name is called.  The name really is, 'The Aged
 Aged Man.'"
 	"Then I ought to have said "That's what the song is called'?"
 Alice corrected herself.
 	"No, you oughtn't:  that's quite another thing!  The song is
-called `Ways and Means':  but that's only what it is called you know!"
+called 'Ways and Means':  but that's only what it is called you know!"
 	"Well, what is the song then?" said Alice, who was by this time
 completely bewildered.
 	"I was coming to that," the Knight said.  "The song really is
@@ -12953,7 +12953,7 @@ The one good thing about repeating your 
 to cringe.
 %
 The only possible interpretation of any research whatever in the
-`social sciences' is: some do, some don't.
+'social sciences' is: some do, some don't.
 		-- Ernest Rutherford
 %
 The only problem with being a man of leisure is that you can never stop
@@ -13008,7 +13008,7 @@ were not for this penalty, the jury woul
 %
 	The people of Halifax invented the trampoline.  During the
 Victorian period the tripe-dressers of Halifax stretched tripe across a
-large wooden frame and jumped up and down on it to `tender and dress'
+large wooden frame and jumped up and down on it to 'tender and dress'
 it.  The tripoline, as they called it, degenerated into becoming the
 apparatus for a spectator sport.
 
@@ -13755,7 +13755,7 @@ said a monk, coming into a teahouse wher
 	"And yet just a short time ago, I was challenged by a scholar
 with an unanswerable question," said Nasrudin.
 	"I could have answered it if I had been there."
-	"Very well.  He asked, `Why are you breaking into my house in
+	"Very well.  He asked, 'Why are you breaking into my house in
 the middle of the night?'"
 %
 There is nothing wrong with Southern California that a rise in the
@@ -15200,7 +15200,7 @@ and get you."
 		-- Jerry Lewis
 %
 When I was crossing the border into Canada, they asked if I had any
-firearms with me.  I said, `Well, what do you need?'
+firearms with me.  I said, 'Well, what do you need?'
 		-- Steven Wright
 %
 When I was in school, I cheated on my metaphysics exam:
@@ -15670,7 +15670,7 @@ Writing about music is like dancing abou
 "Wrong," said Renner.
 
 "The tactful way," Rod said quietly, "the polite way to disagree with
-the Senator would be to say, `That turns out not to be the case.'"
+the Senator would be to say, 'That turns out not to be the case.'"
 %
 X-rated movies are all alike -- the only thing they leave to the
 imagination is the plot.
